src/vivarium_testing_utils/fuzzy_checker.py

```diff
@@ -262,9 +264,9 @@ def _fit_beta_distribution_to_uncertainty_interval(
         mean_min = lower_bound
         mean = (upper_bound + lower_bound) / 2
 
-        best_error = None
```

Comment:
@zmbc I've added you as a reviewer for these minor changes made to this function to appease the mypy overlords.

Reply:
This seems correct (and better than how it was) 👍